### PR TITLE
Join table review

### PR DESCRIPTION
**Pull request by akezeth**
- Added Vets, Specializations and visits table to note vets and their specializations. and the animal visits.
- Added the data insertion as per the requirements.
- Added the required queries.

**1st query**
1st query

**2nd query**
2nd query

**3rd query**
3rd query

**4th query**
4th query

**5th query**
5th query

**6th query**
6th query

**7th query**
7th query

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -129,3 +129,61 @@ from owners join animals on animals.owners_id = owners.id
...
+order by v.visit_date asc
+limit 1;
+
+select a.name as animal_name, v.visit_date, ve.name as vet_name, ve.age as vet_age
+from animals a
+    inner join visits v on a.id = v.animal_id
+    inner join vets ve on v.vet_id = ve.id
+order by v.visit_date desc
+limit 1;
+
```
Comment (rloterh) [inline queries.sql:183]: - [ ] Let's ensure to add the missing query for the following scenario:
  * _What specialty should Maisy Smith consider getting? Look for the species she gets the most._

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Aklilu, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Good job so far implementing join queries!
- ReadMe looks great!

## Required Changes ♻️
- [ ] Let's ensure that the attached screenshots cover all 9 query scenarios.

_Kindly check the other inline comment under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by akezeth** [comment]
- Added query for 
       _What specialty should Maisy Smith consider getting? Look for the species she gets the most._

**1st query**
1st query

**2nd query**
2nd query

**3rd query**
3rd query

**4th query**
4th query

**5th query**
5th query

**6th query**
6th query

**7th query**
7th query

**8th query**
8th query

**9th query**
9th query

**Review by DuaneDave — APPROVED** [review]
Hi @akezeth 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Missing query added
✔️ Good use of gitflow

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**